configs/proactive-scaler/base/deployments.yaml

```diff
@@ -62,35 +62,3 @@ spec:
             requests:
               memory: "45Gi"
               cpu: "10"
----
```

Comment:
@manish-jangra Let's remove 2x large from here since we remove that from p02 and we want to get rid of that from rh01 and p01 too